resources/views/PageDesign/PageDesign.twig

```diff
@@ -197,5 +197,7 @@
 
         </script>
 
+        {{ get_filtered_tags() }}
```

Comment:
Kommen hier auch die Style Tags raus? Falls ja sollten die in den Header auf der höhe wo auch der get_additional_styles() ist rauskommen.

Und die Script Tags wären denke ich auch besser wenn sie über der initialisierung von Vue rauskommen würden also bei dem AfterScriptsLoaded Container dann könnte man dort theoretisch eigene Componenten adden.

Reply:
ich hoffe dass das nicht zu spät ist, nicht dass das schon gerendert wurde bevore der container in Verwendung kommt.
Könnt ihr das anpassen und testen? Ich bin gerade an nem wichtigen Mail Task dran.

Danke

Reply:
nee ich meine damit, dass das ja nicht pro aktiv im controller gefiltert wird, sondern während der container benötigt wird. Quasi während des Twig renderings der kompletten Seite.

Ich weiß daher nicht ob der obere Teil vom PageDesign nicht bereits gerendert wurde. Man müsste es also einmal testen.